brave/browser-laptop
# Navigation icon and style refresh

Fix #13981 

Adjusts much of the top nav / url bar area including:
- Buttons get active state and updated hover state
- Buttons get new icons
- Url bar gets alignment and height change
- Url bar titlemode only swaps the url for the page title - it does not hide the buttons
- Url bar gets a loading progress bar background
- Payments / Funding button gets same states as other navigation buttons: hover, active, disabled, checked (cc @NejcZdovc @ryanml )
- Tab gets shape change
- Tab loading icon is more solid
- Tab page next / prev page navigation button gets new icons
- Tab preview is less "poppy" (and content area is less faded)
- Preferences navigation icons updated

See #13981 for visual details and evolution.

This PR represents the third round of feedback and I feel is sufficient to be merged for the next beta release in order to test other release features with these style changes merged. Subsequent feedback based on that can be covered by a subsequent PR.

## Submitter Checklist:


## Test Plan:
 - Confirm buttons and URL bar work as expected
 - Confirm styles look as expected

## Reviewer Checklist:


Tests
